### PR TITLE
List View for Tasks (Default List)

### PR DESCRIPTION
**Pull request by gaurav613**
- Query to get list and tasks
- "Hardcoded" as of now for one list, will be changed in the future

### CODE REVIEW

File: listView.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="">
+
+
+<head>
```
Comment (Rolo123y) [inline listView.php:5]: This <body> might not be required as we will be having a separate BoardView html page to view the results from this listView page

**Review by Rolo123y — COMMENTED** [review]
The implementation is good as of now. Later, we will need to combine this with a BoardView to actually have a working application. Currently, there is only one list that is being outputted. In the future, there would be multiple outputs of lists, each with their own tasks+Descriptions.

image
An example of what the taskView should aim to look like.

File: listView.php (hunk 1 above)
Comment (gaurav613) [reply listView.php:5]: Yup, I just added this as a placeholder - can be removed once the BoardView page is finalized.

**Review by Rolo123y — APPROVED** [review]
The changes seem good as of now and can be integrated with the BoardView.

**Pull request merged**